search/includes/classes/class-search.php

```diff
@@ -1987,7 +1990,7 @@ public function log_ep_invalid_response( $request, $query, $query_args, $query_o
 		}
 
 		$this->logger->log( 'warning', 'search_query_failure', $message, [
-			'request'    => $request,
+			'request' => $request,
```

Comment:
I think arrow alignment is the official way?

Reply:
Whoops didn't mean to do this.